product-service/serverless.yml

```diff
@@ -12,14 +12,20 @@ provider:
   region: eu-west-1
   httpApi:
     cors: true
+  environment:
+    PG_HOST:
+    PG_PORT:
+    PG_DATABASE:
+    PG_USERNAME:
+    PG_PASSWORD:
```

Comment:
How does it works if you didn't set values?

Reply:
Thanks, added dotenv plugin